### ISSUE TITLE
[Bug]  The accuracy of mae recurrence is too low

### ISSUE THREAD

**Issue by bobo0810** [open-mmlab/mmpretrain#1715]
### 分支

main 分支 (mmpretrain 版本)

## result
The accuracy of mae recurrence is too low
![image](https://github.com/open-mmlab/mmpretrain/assets/26057879/de7af304-e517-4bea-b18e-a74248e6df8a)

## config
The weights and configuration files must be the same as official ones
![image](https://github.com/open-mmlab/mmpretrain/assets/26057879/dbbe905c-42d3-456d-bbb4-3da24a5c1f95)

## log file
[train.log](https://github.com/open-mmlab/mmpretrain/files/12080829/20230717_110512.log)

## dataset
 ILSVRC2012(ImageNet-1k  train_128w  val_5w)  

## code
```python
CONFIG_FILE="configs/mae/benchmarks/vit-base-p16_8xb128-coslr-100e_in1k.py" 
CHECKPOINT="mae_vit-base-p16_8xb512-fp16-coslr-1600e_in1k_20220825-f7569ca2.pth" 
bash ./tools/dist_train.sh ${CONFIG_FILE} ${GPU_NUM} ${NNODES}  \
    --cfg-options model.backbone.init_cfg.checkpoint=${CHECKPOINT}
```

### 环境信息

{'sys.platform': 'linux',
 'Python': '3.8.8 (default, Feb 24 2021, 21:46:12) [GCC 7.3.0]',
 'CUDA available': True,
 'numpy_random_seed': 2147483648,
 'GPU 0,1': 'NVIDIA GeForce RTX 3090',
 'CUDA_HOME': '/usr/local/cuda',
 'NVCC': 'Cuda compilation tools, release 11.7, V11.7.99',
 'GCC': 'gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0',
 'PyTorch': '2.0.1+cu117',
 'TorchVision': '0.15.2+cu117',
 'OpenCV': '4.5.4',
 'MMEngine': '0.7.3',
 'MMCV': '2.0.0rc4',
 'MMPreTrain': '1.0.0rc8+4dd8a86'}

**Comment by Ezra-Yu** [comment]
please add `prefix='backbone'` in this line https://github.com/open-mmlab/mmpretrain/blob/ae7a7b75600923a7384b6c81e515626bf0a3ab4c/configs/mae/benchmarks/vit-base-p16_8xb128-coslr-100e_in1k.py#L60C37-L60C55

**Comment by bobo0810** [comment]
> please add `prefix='backbone'` in this line https://github.com/open-mmlab/mmpretrain/blob/ae7a7b75600923a7384b6c81e515626bf0a3ab4c/configs/mae/benchmarks/vit-base-p16_8xb128-coslr-100e_in1k.py#L60C37-L60C55

prefix='backbone' Does that mean fixed backbone weight ？

**Comment by Ezra-Yu** [comment]
> prefix='backbone' Does that mean fixed backbone weight ？

No，it means using the 'backbone' prefix part of the weight,

checkpoint is a dict with 'head', 'neck', 'backbone'. Since the init_cfg is set in backbone, (`mode.backbone.init_cfg`)

**Comment by bobo0810** [comment]
`--cfg-options model.backbone.init_cfg.checkpoint=${CHECKPOINT}`  
Cause of error： the self-supervised weight is `backbone.a.b`, but the backbone initcfg that recognizes the model does not recognize `backbone.a.b` and only matches `A.B`.
